src/pages/api/auth/[...auth0].ts

```diff
@@ -1,3 +1,4 @@
 import { handleAuth } from "@auth0/nextjs-auth0";
+import { withSentry } from "@sentry/nextjs";
 
-export default handleAuth();
+export default withSentry(handleAuth());
```

Comment:
@sergous N.B.: We need to wrap all new API routes with `withWentry()`.

Reply:
Great tool, I like it